projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/memberships.php

```diff
@@ -208,12 +201,8 @@ public function create_products( $request ) {
 
 		if ( ( defined( 'IS_WPCOM' ) && IS_WPCOM ) ) {
 			require_lib( 'memberships' );
-			$connected_destination_account_id = Jetpack_Memberships::get_connected_account_id();
-			if ( ! $connected_destination_account_id ) {
-				return new WP_Error( 'no-destination-account', __( 'Please set up a Stripe account for this site first', 'jetpack' ) );
-			}
 
-			$result = Memberships_Product::generate_default_products( get_current_blog_id(), $request['type'], $request['currency'], $connected_destination_account_id, $is_editable );
+			$result = Memberships_Product::generate_default_products( get_current_blog_id(), $request['type'], $request['currency'], $is_editable );
```

Comment:
You  probably already know this, but just a heads  up that we may need to deploy WPCOM changes before deploying this to prod

Reply:
Yes, and it has been deployed (and changes on this PR and reflected on the WPCOM one)